File: Day_2/program2.java
import java.util.*;

class program2
{
	public static void main(String A[])
	{
		Scanner sobj = new Scanner(System.in);
		
		System.out.println("Enter the number:");
		int No = sobj.nextInt();
		
		while(No != 0){
			System.out.println(No);
			No = No-1;
		}		
	}
}

File: Day_2/program2.c
#include<stdio.h>

void Display(int No)
{
	while(No != 0)
	{
		printf("%d\n",No);
		No = No-1;
	}
}
int main()

{
	int No = 0;
	printf("Enter the number:");
	scanf("%d",&No);
	Display(No);
	
	return 0;
}
